docs/source/algorithms/ConvertToMD-v1.rst

```diff
@@ -281,6 +288,35 @@ This example produces 3-dimensional dataset, with a temperature axis.
 
    Resulting MD workspace contains 605880 events and 3 dimensions
 
+**Example - Convert to Q-space, indexed version:**
+
+The code iteslef works but disabled from doc tests as takes too long to complete. User should provide its own
+event nexus file instead of **TOPAZ_3132_event.nxs** used within this example. The original **TOPAZ_3132_event.nxs**
+file is available in `Mantid system tests repository <https://github.com/mantidproject/systemtests/tree/master/Data/TOPAZ_3132_event.nxs>`_.
```

Comment:
I think this separate `system test` repository is essentially dead. More accurate to wget `http://mantidweb.nd.rl.ac.uk/externaldata/isis-readonly/md5/{id}` where `{id}` is taken from https://github.com/mantidproject/mantid/blob/master/Testing/Data/SystemTest/TOPAZ_3132_event.nxs.md5 i.e. f3508a6670d300b4c4880c81dbf57bde and then rename to target filename.